### PR TITLE
Production release - upgrade rewards sdk

### PR DESCRIPTION
**Pull request by tackley**
- #70

### CODE REVIEW

**Review by mrose17 — DISMISSED** [review]
Keepin’ it in sync!

File: src/subscriptions.ts [hunk 1]
```diff
@@ -42,36 +42,39 @@ export async function recoverCredsIfRequired(orderId: string): Promise<void> {
...
 export async function checkSubscribedUsingSDK(): Promise<boolean> {
   try {
     const sdk = await loadRewardsSdk();
-    console.log(`brave-rewards-js-sdk: calling credential_summary...`);
+    log(`calling credential_summary...`);
 
     const result = await sdk.credential_summary();
-    console.log("brave-rewards-js-sdk: credential_summary returns", result);
+    log("credential_summary returns", result);
     if (result) {
```
Comment (evq) [inline src/subscriptions.ts:73]: credential_summary now returns even in cases where the credential has expired, you'll want to change this to `result && result.active`

Comment (tackley) [reply src/subscriptions.ts:73]: Oh yes, you did say that. Will update, thanks.

Comment (tackley) [reply src/subscriptions.ts]: updated in #72

**Review by mrose17 — APPROVED** [review]
I approve this PR!

**Pull request merged**